scripts/terraform.sh

```diff
@@ -92,25 +93,37 @@ function destroy() {
 
   init
 
-  local db_inst_name
-  db_inst_name="$(terraform output -json 'en' | jq '. | .db_inst_name' | tr -d \")"
-  # DB often failed to be destroyed by terraform due to "used by other process",
+  # DB always failed to be destroyed by terraform as it's set to not to destroy,
   # so delete it manually
-  gcloud sql instances delete ${db_inst_name} -q --project=${PROJECT_ID} || true
+  local db_inst_names
+  # Fetching databases from previous terraform deployment output is not always reliable,
+  # especially when previous terraform deployment failed.
+  # If a user invoke `destroy` function here, it should be safe to delete all databases
+  # prefix with `en-verification`in this project.
+  db_inst_names="$(gcloud sql instances list --project=${PROJECT_ID} | grep -Eo 'en-verification-[a-zA-Z0-9]+' || true)"
+  if [[ -n "${db_inst_names}" ]]; then
+    for db_inst_name in ${db_inst_names}; do
+      gcloud sql instances delete ${db_inst_name} -q --project=${PROJECT_ID}
```

Comment:
I'm... worried about this. If we (or someone else) accidentally ran this in prod, it'd be disastrous.

Reply:
The alternative approach is to grab the db instance name from terraform state, how does that sound to you?

Reply:
Safer 😄

Reply:
Updated to fetch from terraform state, PTAL

Reply:
@sethvargo ^^